karapace/schema_registry_apis.py

```diff
@@ -16,11 +18,17 @@
 from typing import Any, Dict, NoReturn, Optional, Tuple
 
 import aiohttp
+import aiohttp.web
 import async_timeout
 import asyncio
+import crypt
```

Comment:
Crypt will be deprecated in Python 3.11. Recommendation is to use `hashlib` instead.